### DISCUSSION TITLE
iTwinUI-react has moved

### DISCUSSION THREAD

**Discussion by mayank99** [iTwin/iTwinUI-react#1001] [Announcement]
***TL;DR: The code for iTwinUI-react has been moved to the main iTwinUI repo at [github.com/iTwin/iTwinUI](https://github.com/iTwin/iTwinUI).***

---

This repo has been archived and will no longer be updated. The contents of this repo have moved to  [iTwin/iTwinUI](https://github.com/iTwin/iTwinUI/tree/main/packages/itwinui-react). This includes all the code, open issues, wiki pages, storybook, etc.
- code for iTwinUI-react: https://github.com/iTwin/iTwinUI/tree/main/packages/itwinui-react
- storybook demos (github-pages): https://itwin.github.io/iTwinUI
- issues: https://github.com/iTwin/iTwinUI/issues
- discussions: https://github.com/iTwin/iTwinUI/discussions
- pull requests: https://github.com/iTwin/iTwinUI/pulls
- wiki: https://github.com/iTwin/iTwinUI/wiki

This means you will not be able to create new issues/discussions/PRs or respond to existing ones. Any existing links (e.g. to wiki or storybook) will continue to work but will not receive updates. For the most up-to-date information or code, please visit https://github.com/iTwin/iTwinUI.

This was done to improve both the developer experience and user experience. You can find a more detailed rationale at https://github.com/iTwin/iTwinUI/issues/870.
